package.json

```diff
@@ -0,0 +1,56 @@
+{
+	"name": "trader",
+	"version": "0.0.1",
```

Comment:
In my experience with deployment, we need to include a bigger builder number in this version number that auto-increments. Doesn't need to be now, but it'll become important when we start passing it around.

Reply:
I think this will be out of the scope of this issue so that this doesn't block the other tasks anymore. I raised an issue for auto-incrementing our versions in general: https://github.com/agriculture-capstone/systems/issues/2